### PR TITLE
Updates for new Japanese calendar era

### PR DESCRIPTION
**Pull request by rpetrusha**
## Updates for new Japanese calendar era

Related to dotnet/samples#686

Contributes to dotnet/docs#10192

Depends on dotnet/docs#10853

//cc @tarekgh

### CODE REVIEW

File: includes/calendar-era.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
...
-
+>  A new era in the <xref:System.Globalization.JapaneseCalendar> and <xref:System.Globalization.JapaneseLunisolarCalendar> begins on May 1, 2019. This change affects all applications that use these calendars. See [Handling a new era in the Japanese calendar in .NET](https://blogs.msdn.microsoft.com/dotnet/2018/11/14/handling-a-new-era-in-the-japanese-calendar-in-net/) for more information and to determine whether your applications are affected. See [Prepare your application for the Japanese era change](/windows/uwp/design/globalizing/japanese-era-change) for information on testing your applications on Windows systems to ensure their readiness for the era change. See the ["Working with eras"](~/docs/standard/datetime/working-with-calendars.md#working-with-eras) section of the [Working with calendars](~/docs/standard/datetime/working-with-calendars.md) article for additional information and recommendations for using eras in the Japanese calendars.
```
Comment (tarekgh) [inline includes/calendar-era.md]: The link (/windows/uwp/design/globalizing/japanese-era-change)  looks not working. is this a place holder?

Comment (tarekgh) [reply includes/calendar-era.md]: other links looks not working too. I may be missing something but wanted to tell.

Comment (rpetrusha) [reply includes/calendar-era.md]: @tarekgh, the links work in live builds but not in review builds. Any content that originates outside of the dotnet/docs repo isn't build as part of this PR, so Windows content isn't built.

**Comment by rpetrusha** [comment]
Closing and reopening to begin new build after dotnet/samples#686 was merged.

**Comment by rpetrusha** [comment]
Closing and reopening to begin new build before merge.

**Pull request merged**